.github/workflows/gradle.yml

```diff
@@ -6,6 +6,7 @@ on:
   pull_request:
     branches:
       - master
+      - 2.5.x
```

Comment:
This was already included in `master` branch and it allows to run tests for PRs to 2.5.x